notebooks/official/experiments/get_started_with_vertex_experiments_autologging.ipynb

```diff
@@ -0,0 +1,1230 @@
+{
```

Comment:
<p>Data science -&gt; data science</p><p>you may want -&gt; you</p><p>Vertex Experiments -&gt; Vertex AI Experiments</p><p>Experiment Runs -&gt; experiment runs</p><p> </p><p>End the Overview with a tag/linkback.</p><p> </p><p><code>Learn more about \[Autologging data to an experiment run\](</code>https://cloud.google.com/vertex-ai/docs/experiments/autolog-data<code>)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>